packages/blockchain-wallet-v4-frontend/src/providers/TranslationsProvider/index.js

```diff
@@ -8,7 +8,7 @@ import { selectors } from 'data'
 
 class TranslationsProvider extends React.Component {
   state = {
-    locale: '',
+    locale: 'en',
```

Comment:
setting this to be default language, we have init function which is not called at all.
We do call this [saga](https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/data/rootSaga.js#L41-L54) after we tried to get react-intl stuff. 

However react-intl have some upgrades to version 5, we are still using v2, it might be worth to migrate to latest, I opened a tech debt ticket for it